src/foundation/Icon/Generated/IconBell12.tsx

```diff
@@ -11,21 +11,12 @@ function IconBell12(props: GeneratedIconProps): React.ReactElement<GeneratedIcon
             name="IconBell12"
             {...props}
         >
-            <g fill="none">
```

Comment:
Mmmm, Why a lot of size 12 icons got updated? That `<g fill="none"` may cause an issue in Clarify with a possibility of the icons going blank.

Reply:
No clue! I just pulled the branch and ran the generate icons command...